### PR TITLE
Test errors?

### PR DESCRIPTION
**Pull request by maelle**

### CODE REVIEW

File: R/utils.R [hunk 1]
```diff
@@ -233,7 +233,7 @@ getResults_bypage <- function(urlAQ, argsList){
     Sys.sleep(60*5+5)
     res <- client$get(query = argsList)
   }
-  while(res$status_code >= 400 && try_number < 6) {status <- get_status()
+  while(res$status_code >= 400 && try_number < 2) {status <- get_status()
```
Comment (maelle) [inline R/utils.R]: this is just for making tests faster for now.

**Comment by sckott** [comment]
Looks good to me. 

(note that retry verb might be something to look into as well in the case of 429's https://docs.ropensci.org/crul/reference/HttpClient.html#method-retry)

**Comment by maelle** [comment]
starting a new branch

**Comment by maelle** [comment]
starting a new branch

**Pull request closed without merging**